src/lib/viewers/BaseViewer.js

```diff
@@ -1030,6 +1031,11 @@ class BaseViewer extends EventEmitter {
             return this.viewerConfig.enabled;
         }
 
+        // Disable new annotations for Excel and iWork formats
+        if (EXCEL_EXTENSIONS.includes(extension) || IWORK_EXTENSIONS.includes(extension)) {
+            this.options.showAnnotationsControls = false;
```

Comment:
Do we want to mutate the options object in here? Seems like we should just `return false`. Do we need a new method for the new experience?